src/main/java/net/javadiscord/javabot/systems/moderation/AutoMod.java

```diff
@@ -85,6 +90,8 @@ private boolean canBypassAutomod(Member member) {
 	 * @param message the {@link Message} that should be checked
 	 */
 	private void checkNewMessageAutomod(@Nonnull Message message) {
+		AtomicBoolean consideredSpam = new AtomicBoolean(false);
```

Comment:
Why is this needed? (also applies to the usage at the bottom)

Reply:
Because of the use in the lambda

Reply:
Because its not that useful if the bot spams the embed for not enough time and spam, since this would be spam again. My idea was that when it was already considered as spam its not necessary to send the other embed too.